src/services/quota/quota.test.ts

```diff
@@ -319,95 +291,6 @@ describe("quota", () => {
     });
   });
 
-  describe("postTransaction", () => {
```

Comment:
refactored this out as it was exhausting to go through everything that is called "quota"

Reply:
Thanks for doing this!